src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -341,11 +344,6 @@ def pix2deg(self) -> None:
 
         After success, the gaze dataframe is extended by the resulting dva position columns.
 
-        Raises
```

Comment:
We shouldn't remove this.

Reply:
pydoclint complains that the function does not have a raise in it, it just stupidly checks for raises within the scope of function :-( -- ignore the pydoclint error?